src/main/java/com/greghaskins/spectrum/internal/NameSanitiser.java

```diff
@@ -4,18 +4,20 @@
 import java.util.Set;
 
 /**
- * Santises names within a Suite. To stop a name being duplicated or
+ * Sanitises names within a Suite. To stop a name being duplicated or
  * containing characters that upset test runners.
  */
 public class NameSanitiser {
   private Set<String> namesUsed = new HashSet<>();
 
   /**
-   * Given a name, deduplicate it and filter out any bad characters.
+   * Deduplicate the given {@code name} and filter out any bad characters.
+   *
+   * <p>Note: this function has side effects - sanitising a name will cause it to be remembered for future
+   * deduplication purposes.
+   *
    * @param name the spec name
-   * @return a name which is non-duplicate within this sanitiser and which has known
-   *        bad characters removed. Note: this function has side effects - sanitising
-   *        a name will cause it to be remembered for future deduplication purposes.
+   * @return a name unique to this sanitiser which has known bad characters removed.
```

Comment:
`Style` `Minor`

> The typo was in the class-level Javadoc but I thought I'd clean up this method-level Javadoc while I was there.

I prefer to have short `@param` and `@return` clauses with clarifying remarks and other notes captured in the body of the method-level Javadoc. I find it easier to read such short clauses at a glance.

In this specific case the fact that the method has side-effects is a comment about the method as a whole and does not apply solely to the return value: as such I snipped it from the return clause into the body of the method-level Javadoc.